components/CollectionCells/src/MDCCollectionViewCell.m

```diff
@@ -255,6 +258,22 @@ - (CGRect)accessoryFrame {
   return CGRectMake(originX, originY, size.width, size.height);
 }
 
+- (MDCInkView *)inkView {
+  if (!_inkView) {
+    _inkView = [[MDCInkView alloc] initWithFrame:self.bounds];
+    [self addSubview:_inkView];
+  }
+  return _inkView;
+}
+
+- (void)setInkView:(MDCInkView *)inkView {
+  if (_inkView) {
+    [_inkView removeFromSuperview];
+  }
+  [self addSubview:inkView];
```

Comment:
I know we are just moving existing code, but this will break if inkView is nil since addSubview requires nonnull.

Consider:
if (_inkView) {
  [_inkView removeFromSuperView];
  _inkView = nil;
}
if (inkView) {
  [self addSubview:inkView];
  _inkView = inkView;
}